src/configure_transform.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+
+import os
+import re
+import sys
+
+define_re = re.compile("#define ([A-Z_]+) (.*)")
+
```

Comment:
maybe just a bug for [ipython](https://bugzilla.redhat.com/show_bug.cgi?id=874133)? We can always test it out.